stub-server/src/main/java/org/zanata/rest/service/MockFileResource.java

```diff
@@ -61,6 +61,7 @@
 @Path(FileResource.SERVICE_PATH)
 public class MockFileResource implements FileResource {
     @Override
+    @Deprecated
```

Comment:
Well, if `FileResource` has a javadoc with a suitable `@deprecated` tag, that can be inherited (as long as the children don't replace the parent javadoc).